### PR TITLE
Rncamera migration

### PR DESCRIPTION
**Pull request by jgfidelis**
iOS is working fine (in example).

Android rncamera example is not starting 

```
02-02 18:35:20.409 31142-31142/com.rncameraexample E/AndroidRuntime: FATAL EXCEPTION: main
                                                                     Process: com.rncameraexample, PID: 31142
                                                                     java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object java.util.SortedSet.last()' on a null object reference
                                                                         at com.google.android.cameraview.Camera1.adjustCameraParameters(Camera1.java:475)
                                                                         at com.google.android.cameraview.Camera1$1.onSurfaceChanged(Camera1.java:108)
                                                                         at com.google.android.cameraview.PreviewImpl.dispatchSurfaceChanged(PreviewImpl.java:56)
                                                                         at com.google.android.cameraview.TextureViewPreview$1.onSurfaceTextureAvailable(TextureViewPreview.java:45)
                                                                         at android.view.TextureView.getHardwareLayer(TextureView.java:370)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15443)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at android.view.View.draw(View.java:16539)
                                                                         at android.widget.FrameLayout.draw(FrameLayout.java:598)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15466)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at com.facebook.react.views.view.ReactViewGroup.dispatchDraw(ReactViewGroup.java:778)
                                                                         at android.view.View.draw(View.java:16539)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15466)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15461)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15461)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15461)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.View.draw(View.java:16258)
                                                                         at android.view.ViewGroup.drawChild(ViewGroup.java:3713)
                                                                         at android.view.ViewGroup.dispatchDraw(ViewGroup.java:3506)
                                                                         at android.view.View.draw(View.java:16539)
                                                                         at android.widget.FrameLayout.draw(FrameLayout.java:598)
                                                                         at com.android.internal.policy.impl.PhoneWindow$DecorView.draw(PhoneWindow.java:3111)
                                                                         at android.view.View.updateDisplayListIfDirty(View.java:15466)
                                                                         at android.view.View.getDisplayList(View.java:15488)
                                                                         at android.view.ThreadedRenderer.updateViewTreeDisplayList(ThreadedRenderer.java:309)
                                                                         at android.view.ThreadedRenderer.updateRootDisplayList(ThreadedRenderer.java:315)
                                                                         at android.view.ThreadedRenderer.draw(ThreadedRenderer.java:354)
                                                                         at android.view.ViewRootImpl.draw(ViewRootImpl.java:2956)
                                                                         at android.view.ViewRootImpl.performDraw(ViewRootImpl.java:2753)
                                                                         at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2339)
                                                                         at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1314)
                                                                         at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7057)
                                                                         at android.view.Choreographer$CallbackRecord.run(Choreographer.java:829)
                                                                         at android.view.Choreographer.doCallbacks(Choreographer.java:606)
                                                                         at android.view.Choreographer.doFrame(Choreographer.java:576)
                                                                         at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:815)
                                                                         at android.os.Handler.handleCallback(Handler.java:739)
                                                                         at android.os.Handler.dispatchMessage(Handler.java:95)
                                                                         at android.os.Looper.loop(Looper.java:145)
                                                                         at android.app.ActivityThread.main(ActivityThread.java:6934)
                                                                         at java.lang.reflect.Method.invoke(Native Method)
                                                                         at java.lang.reflect.Method.invoke(Method.java:372)
                                                                     	at com.android
```

### CODE REVIEW

File: android/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="com.lwansbrough.RCTCamera">
+<manifest xmlns:android="http://schemas.android.com/apk/res/android" package="org.reactnative">
```
Comment (sibelius) [inline android/src/main/AndroidManifest.xml]: nice

**Comment by EskelCz** [comment]
I've ran into this issue on 2.3.0 with Xiomi RedMi 6.
@jgfidelis Any idea why this might still happen? Thanks

**Comment by EskelCz** [comment]
react-native-camera uses android library camera view, which as this issue open.
seems like it's a rare occurrence and doesn't get much attention.
https://github.com/google/cameraview/issues/182

**Comment by sibelius** [comment]
can you try with latest version?

**Comment by EskelCz** [comment]
@sibelius I'm confused, 2.3.0 is the latest version, is it not?
https://github.com/react-native-community/react-native-camera/releases

**Comment by sibelius** [comment]
open a new issue with your problem

**Pull request merged**